custom_components/daikin_onecta/const.py

```diff
@@ -252,7 +254,7 @@
         ENTITY_CATEGORY: EntityCategory.DIAGNOSTIC,
     },
     "isFirmwareUpdateSupported": {
-        CONF_DEVICE_CLASS: None,
+        CONF_DEVICE_CLASS: BinarySensorDeviceClass.UPDATE,
```

Comment:
I think my suggestion about the update part is wrong. This one should remain as None. The sensor only states whether updating is supported.